satellite/console/service.go

```diff
@@ -657,15 +663,14 @@ func (s *Service) GetUsersProjects(ctx context.Context) (ps []Project, err error
 }
 
 // GetCurrentRewardByType is a method for querying current active reward offer based on its type
-func (s *Service) GetCurrentRewardByType(ctx context.Context, offerType rewards.OfferType) (reward *rewards.Offer, err error) {
+func (s *Service) GetCurrentRewardByType(ctx context.Context, offerType rewards.OfferType) (offer *rewards.Offer, err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	reward, err = s.rewards.GetCurrentByType(ctx, offerType)
+	offers, err := s.rewards.GetActiveOffersByType(ctx, offerType)
 	if err != nil {
 		return nil, errs.New(internalErrMsg)
```

Comment:
We would need to log this error too, however, this could be done in another time.